From a25634240a19d9a0efe1391f60aa699687579c26 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:28 -0400
Subject: [redhat] kabi: Adding symbol param_ops_bool (kernel/params.c)

Message-id: <f6b84d6bc3b430e6de0132a8be562d8d25b0f1ed.1410510773.git.poros@redhat.com>
Patchwork-id: 95945
O-Subject: [PATCH RHEL7.1 148/233] kabi: Adding symbol param_ops_bool (kernel/params.c)
Bugzilla: 1062361
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361

    Description:
    ============
    Not used by PowerPath directly.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/param_ops_bool
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/param_ops_bool
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/param_ops_bool

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/param_ops_bool b/redhat/kabi/kabi-module/kabi_ppc64/param_ops_bool
new file mode 100644
index 0000000..5bcf049
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/param_ops_bool
@@ -0,0 +1,2 @@
+#1-
+0x1976aa06 param_ops_bool vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/param_ops_bool b/redhat/kabi/kabi-module/kabi_s390x/param_ops_bool
new file mode 100644
index 0000000..5bcf049
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/param_ops_bool
@@ -0,0 +1,2 @@
+#1-
+0x1976aa06 param_ops_bool vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/param_ops_bool b/redhat/kabi/kabi-module/kabi_x86_64/param_ops_bool
new file mode 100644
index 0000000..5bcf049
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/param_ops_bool
@@ -0,0 +1,2 @@
+#1-
+0x1976aa06 param_ops_bool vmlinux EXPORT_SYMBOL
-- 
1.7.1